recipe/build.sh

```diff
@@ -1,20 +1,28 @@
 #!/bin/bash
 
+if [[ $target_platform == linux* ]]; then
+  ln -s -t "${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64/" $PREFIX/lib/libexpat*
+fi
+
 [[ -d build ]] || mkdir build
 cd build
 
+# Missing g++ workaround.
+ln -s ${GXX} g++ || true
+chmod +x g++
+export PATH=${PWD}:${PATH}
+
```

Comment:
I used QMAKE_CXX=${CXX} to fix this in #19 which is a bit cleaner

Reply:
Thanks! Testing that here now.